frontend/src/components/FileList.vue

```diff
@@ -14,8 +14,12 @@
               />
             </div>
           </th>
-          <th scope="col" class="text-left">Name</th>
-          <th scope="col" class="text-left">Last Modified</th>
+          <th scope="col" class="text-left">
+            Name<img :src="getOrderIcon('name')" @click="reorderBy('name')" width="16" height="16">
```

Comment:
Move the click event to the `<th>` because currently you have to click the small icons directly to change the sorting
Sharepoint also uses the whole cell for the click event.

Otherwise wrap everything into a `<span>` or `<div>` (so name + icon) so you can at least click on the column name to fix sorting